packages/sources/gemini/src/endpoint/reserves.ts

```diff
@@ -11,6 +11,8 @@ export interface ResponseSchema {
 
 export const inputParameters: InputParameters = {
   token: ['token', 'asset', 'coin'],
+  network: false,
```

Comment:
do we have tests to make sure that the adapter works when these fields are missing from the request params?

Reply:
the integration test example works without 'network' and 'chainId'